### DISCUSSION TITLE
How to delete multiple Obj  to JSON-Server in RTK Query?

### DISCUSSION THREAD

**Discussion by HypeNeutron** [reduxjs/redux-toolkit#2317] [Q&A] [answered]
![code](https://user-images.githubusercontent.com/40451633/168001050-5671f984-d11c-4fe6-9ffb-5ba61a549dd5.png)

**Comment by phryneas** [comment, marked as answer]
Is this possible with JSON-server with a single request? If yes, how? I can answer the RTKQ-side but honestly I don't have the time to research a third-party-tool right now.

**Reply by HypeNeutron** [reply]
Thank you, I found it , I jus forgot  that with single request but multiple time, forEach mutation function trigger that it's work
